### ISSUE TITLE
Windows Install Error

### ISSUE THREAD

**Issue by Captain-Wiggles** [XDGFX/ultrasonics#27]
Hello, I am trying to install ultrasonics on a windows 10 machine. I have Python 3.9 installed.

The error I am getting is as follows:

>>> "C:\Program Files (x86)\ultrasonics\app.py"
  File "<stdin>", line 1
    "C:\Program Files (x86)\ultrasonics\app.py"
                                               ^
SyntaxError: (unicode error) 'unicodeescape' codec can't decode bytes in position 22-23: truncated \uXXXX escape
>>>

I opened up python and dragged and dropped the app.py file into it to get this error. I'm not particularly savvy with python and linux to windows installs so any guidance would be appreciated!

**Comment by XDGFX** [comment]
You've opened a Python interactive commandline (where you can type Python commands to run one at a time), and are trying to run the program from inside that. That's not actually the right way to do it 😉, instead you need to tell Python to run the whole file.

I'm not super familiar with Python on Windows, but if you open a Command Prompt (cmd.exe) and type `python3` does it come up with the same interactive session you had before? Or does it display an error like `command not found`? You might need to try a combination, like `python`, `Python`, or `Python3` etc.

If that works, run `python3 C:\Program Files (x86)\ultrasonics\app.py`. You might need to escape the backslashes by doing `python3 C:\\Program Files (x86)\\ultrasonics\\app.py`.

Let me know how you get on!

**Comment by Captain-Wiggles** [comment]
Hey! Thanks for quick explanation. I did exactly as you suggested and it worked! I'll record the process I went through in case anyone runs into the same newbie trouble hehe. 

windows cmd python3.
windows store pops up to install python3.9 for windows. Install python thru windows store

in cmd.exe cd to ultrasonic directory. run "app.py" 

if errors pop up, take a look at what errors there are. 
(i had to install all the plugins that were listed in the "requirements.txt" file)
once all required plugin files were installed, i ran app.py again. 

go to localhost:5000 on browser to access webapp.

Thanks again!
